appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/module/RarHandler.java

```diff
@@ -160,4 +160,13 @@ public List<URI> getClassPathURIs(ReadableArchive archive) {
         }
         return uris;
     }
+
+
+    private static File toFile(URI uri) {
+        try {
+            return new File(uri.getSchemeSpecificPart());
+        } catch (IllegalArgumentException e) {
```

Comment:
Previously used [File(URI)](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/io/File.html#%3Cinit%3E(java.net.URI)) declares `IAEx`, but [File(String)](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/io/File.html#%3Cinit%3E(java.lang.String)) does not.
Can you show, the best with test(s) for this `toFile`, case with `IAEx` being thrown here, please?

Reply:
Good catch, I have seen the exception, then did some incorrect fix, then this, but I forgot to remove it.